CafeMenuEditor/mainwindow.cpp

```diff
@@ -110,6 +112,18 @@ void MainWindow::example()
     refreshMenu();
 }
 
+void MainWindow::aboutMenu()
+{
+    QMessageBox msg;
+    msg.information(this, "Menu Editor", "v.1.0", QMessageBox::Ok);
+}
+
+void MainWindow::aboutQt()
+{
+    QMessageBox msg;
+    msg.aboutQt(this, "about Qt");
```

Comment:
QMessageBox::aboutQt() is a **static** method. So, you can use it without creating a new object.